nextcloud/android
# Feature Open Calendar Events and Contacts

Needs https://github.com/nextcloud/server/pull/46464



**How to Test?**

1. Install [DAVx⁵](https://f-droid.org/en/packages/at.bitfire.davdroid/) and sync via Nextcloud account
2. Search calendar event or contact and give permission to read calendar event and contact, then press event app should redirect to internal app instead of web app.

**How to Test?**

1. Don't give permission then try to open calendar event or contact, app should redirect to the web app.

**Demo**

https://github.com/nextcloud/android/assets/67455295/1cfadfd3-e142-409f-abdd-6d85b3b5ffa5

**Duplicated Contacts**

https://github.com/user-attachments/assets/2d333570-c39d-4567-b188-53252ffb7b50
